app/iCubTest/test_left_arm.ini

```diff
@@ -5,7 +5,7 @@ portname /${robotname}/left_arm
 joints 16
 
 target  -30.0 25.0 10.0 40.0 -10.0 -10.0 10.0 50 15.0 15.0 15.0 15.0 15.0 15.0 15.0 15.0
-min      -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0 -3.0
+min       3.0  3.0  3.0  3.0  3.0  3.0  3.0  3.0  3.0  3.0  3.0  3.0  3.0  3.0  3.0  3.0
```

Comment:
uhm, no. I thought it would be better to specify unsigned thresholds. Does it make sense?